package-lock.json

```diff
@@ -94,9 +94,9 @@
       }
     },
     "@angular/animations": {
-      "version": "7.2.15",
-      "resolved": "https://registry.npmjs.org/@angular/animations/-/animations-7.2.15.tgz",
```

Comment:
This `package-lock.json` file doesn't need to be updated since there is no `@types/lodash` in here... could you remove this file from this PR?